src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -402,6 +412,12 @@ export const WorkspaceForm = ({
                     features.length > 0 ? `(${selectedIds.length}/${features.length})` : ''
                   }`}
                   checked={selectedIds.length > 0}
+                  disabled={
```

Comment:
I don't think it's an issue. Currently there are only two features needed to be checked on default and we will judge them by id `DEFAULT_CHECKED_FEATURES_IDS`. FeatureGroup is different from feature, it needs to be distinguished by name instead of id. 
At present, it seems that only features are non-configurable. I am not sure that there will be featureGroups that need to be selected by default in the future. If we want to add support to featureGroup, we need to update `EuiCheckboxGroup` even though we don't use it at all, do you think we should support it now?

Reply:
Not to support feature group. Currently the `DEFAULT_CHECKED_FEATURES_IDS` will be failed if one feature id is inside a group.

Reply:
Why?  I have this judge logic in https://github.com/ruanyl/OpenSearch-Dashboards/pull/106/files#diff-41a477c4885f8f444ab080e457bb39fcac5787db19db2018e114a0f42e6b0d73R416.  I think we shouldn't support feature in feature group or a feature group now. Because the feature group is judged by name, we cannot know whether the feature group or the features in the group need to be cheched by default in the future. We can determine how to be compatible only when `DEFAULT_CHECKED_FEATURES_IDS` is appended in the future, right?